scenarios/14_Hordes_Undead.cfg

```diff
@@ -234,19 +234,28 @@
         [filter]
             type=Gargoyle
         [/filter]
-        [message]
-            id=Malakar
-            message= _ "They fight with statues?"
-        [/message]
-        [message]
-            id=Kogw
-            message= _ "I've heard stories of them; step carefully, this is powerful black magic."
-        [/message]
-        [message]
-            id=Theracar
-            image=$theracar_image
-            message= _ "I recall rumors of them from my time serving that wretched Baron; this is powerful black magic."
-        [/message]
+        [if]
+            [variable]
+                name=last_b_scenario
+                #Gargoyles show up in 6B, in which case Malakar is already familiar with them
+                less_than=6
+            [/variable]
+            [then]
+                [message]
+                    id=Malakar
+                    message= _ "They fight with statues?"
+                [/message]
+                [message]
+                    id=Kogw
+                    message= _ "I've heard stories of them; step carefully, this is powerful black magic."
+                [/message]
+                [message]
+                    id=Theracar
+                    image=$theracar_image
+                    message= _ "I recall rumors of them from my time serving that wretched Baron; this is powerful black magic."
+                [/message]
+            [/then]
```

Comment:
Maybe an `[else]` branch here, like "These statues again?" or something? I dunno...

Reply:
actually that can wait for a later release; adding such a branch in this PR would give @k-konami extra strings to translate, which probably isn't worth holding up this release for...

Reply:
Japanese users are used to untranslated material, 
so it's okay if there are some omissions.   :) 

The latest POT(Created on 08-16 19:15 UTC") I checked and there were 37 untranslated.
For the time being, I would be grateful if you could merge the ja.po that is currently Pull-Req'd.